app/src/main/java/com/example/bookwormadventuresdeluxe2/LoginActivity.java

```diff
@@ -89,4 +147,177 @@ public void onClick(View view)
+
+                                /* Get User information from firestore and take to MyBooksActivity */
+                                collectionReference
+                                        .whereEqualTo("userId", currentUserId)
+                                        .addSnapshotListener(new EventListener<QuerySnapshot>()
+                                        {
+                                            @Override
+                                            public void onEvent(@Nullable QuerySnapshot queryDocumentSnapshots, @Nullable FirebaseFirestoreException error)
+                                            {
+                                                if (error != null)
+                                                {
+                                                    return;
+                                                }
+                                                if (!queryDocumentSnapshots.isEmpty())
+                                                {
+                                                    for (QueryDocumentSnapshot snapshot : queryDocumentSnapshots)
+                                                    {
+                                                        // add to UserCredentialAPI to be accessible throughout app
+                                                        UserCredentialAPI userCredentialAPI = UserCredentialAPI.getInstance();
+                                                        userCredentialAPI.setUsername(snapshot.getString("username"));
+                                                        userCredentialAPI.setUserId(snapshot.getString("userId"));
+                                                    }
+                                                    progressBar.setVisibility(View.INVISIBLE);
+                                                    // Go to ListActivity
+                                                    startActivity(new Intent(LoginActivity.this, MyBooksActivity.class));
+                                                }
+                                            }
+                                        });
+                            }
+                            else
+                            {
+
+                                /* Set EditText Error type from errorCode */
+                                try
+                                {
+                                    /* Extract Firebase Error Code */
+                                    String errorCode = "";
+                                    errorCode = ((FirebaseAuthException) task.getException()).getErrorCode();
+
+                                    switch (errorCode)
+                                    {
+                                        case "ERROR_WRONG_PASSWORD":
+                                            EditTextValidator.wrongPassword(editTextPassword);
+//                                            progressBar.setVisibility(View.INVISIBLE);
```

Comment:
is there a reason for this?

Reply:
oh forgot to remove this. thanks for noticing